helm/dpga-compliance-barebone/operations-app/templates/deployment.yaml

```diff
@@ -0,0 +1,87 @@
+{{- if .Values.enabled -}}
+
+apiVersion: apps/v1
```

Comment:
these are duplicate files. The beauty of  HELM chart is that you can use an existing chart as a  dependency instead of copying it